### ISSUE TITLE
Not loading on Lenovo X130e

### ISSUE THREAD

**Issue by zaivala** [feren-OS/feren-OS-Bug-Reporting#16]
I quite easily and successfully loaded ferenOS on my Lenovo T430, but when I went to load it on my wife's Lenovo X130e it could not find the wifi. Is it possible that the wifi chip/adapter on the X130e is incompatible? She really wanted to have me install it, but no go.  (Her machine previously had LinuxMint 18.1 on it, and I just successfully installed Ubuntu MATE 16.10 after failing to install ferenOS.)

**Comment by feren** [comment]
@zaivala Hmm, was LM 18.1 able to connect to Internet?

**Comment by zaivala** [comment]
Yep. Well, LM 17.1 was. And she is now running Ubuntu MATE 16.10. LM and Ubuntu did not have the problem, ferenOS did. Strange, as ferenOS is based on LM...which is based on Ubuntu...

**Comment by feren** [comment]
Yeah, that's strange... :/

**Comment by feren** [comment]
@zaivala Can you try explicitly Linux Mint 18.1?

**Comment by zaivala** [comment]
LinuxMint 18.1 was working on the machine. Right now it is running Ubuntu MATE. I don't know what the problem was -- I have an identical machine and it loaded fine on that. Note that I'm a bit old-school and burned ferenOS to a DVD, and am loading it from the drive. And it is the same DVD I used to install to my machine.

**Comment by zaivala** [comment]
I want to report that I successfully installed ferenOS last night and this afternoon.

The T430 in question was shipped to me with a battery which was not recognized as authentic. While waiting for a new battery to arrive (the first one also did not work, and I learned what to look for), I put my laptop battery in my wife's new laptop. Last night I decided to see if that made the difference, and it did. She is VERY happy with her new ferenOS installation. I spent a bit of time installing our favorites (Mullvad VPN, SoftMaker Office, Grub customizer, and Brother MFC-J470DW drivers) and it is now up to speed. Thanks for a fantastic distro!

I would like to state that, like many others have reported for other distros, Cinnamon crashes a lot more than I'd like. I haven't had that problem with MATE, or Budgie, or Enlightenment... It crashed two or three times while installing and updating SoftMaker Office.
